<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="8" hostname="CO-IT026200" failures="1" tests="4" name="Command line test" time="27.247" errors="0" timestamp="2022-11-26T21:40:21 COT">
  <testcase classname="org.bank.test.DuplicateEmailAccountsTest" name="avoidDuplicateEmailsTest" time="1.718"/>
  <testcase classname="org.bank.test.EmptyEndpointTest" name="emptyEndpointTest" time="19.951">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at io.restassured.path.json.JsonPath.getList(JsonPath.java:391)
at org.bank.test.BaseTest.getAllUsers(BaseTest.java:72)
at org.bank.test.BaseTest.emptyTheEndpoint(BaseTest.java:86)
at org.bank.test.EmptyEndpointTest.emptyEndpointTest(EmptyEndpointTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading 'O', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 45 more
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- emptyEndpointTest -->
  <testcase classname="org.bank.test.PostUsersTest" name="createTenUsersTest" time="4.107"/>
  <testcase classname="org.bank.test.UpdateAccountNumberTest" name="updateUserAccountNumberTest" time="1.021"/>
  <testcase classname="org.bank.test.BaseTest" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="org.bank.test.BaseTest" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="org.bank.test.BaseTest" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="org.bank.test.BaseTest" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="org.bank.test.BaseTest" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="org.bank.test.BaseTest" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="org.bank.test.BaseTest" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="org.bank.test.BaseTest" name="@BeforeTest setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setup -->
</testsuite> <!-- Command line test -->
